tests/conftest.py

```diff
@@ -200,56 +201,58 @@ def sushi_context_fixed_date(mocker: MockerFixture) -> Context:
 
 
 @pytest.fixture()
-def sushi_context(mocker: MockerFixture) -> Context:
-    context, plan = init_and_plan_context("examples/sushi", mocker)
+def sushi_context(init_and_plan_context: t.Callable) -> Context:
+    context, plan = init_and_plan_context("examples/sushi")
     context.apply(plan)
     return context
 
 
 @pytest.fixture()
-def sushi_dbt_context(mocker: MockerFixture) -> Context:
-    context, plan = init_and_plan_context("examples/sushi_dbt", mocker)
+def sushi_dbt_context(init_and_plan_context: t.Callable) -> Context:
+    context, plan = init_and_plan_context("examples/sushi_dbt")
 
     context.apply(plan)
     return context
 
 
 @pytest.fixture()
-def sushi_test_dbt_context(mocker: MockerFixture) -> Context:
+def sushi_test_dbt_context(init_and_plan_context) -> Context:
     from tests.fixtures.dbt.sushi_test.seed_sources import init_raw_schema
 
-    context, plan = init_and_plan_context("tests/fixtures/dbt/sushi_test", mocker)
+    context, plan = init_and_plan_context("tests/fixtures/dbt/sushi_test")
     init_raw_schema(context.engine_adapter)
 
     context.apply(plan)
     return context
 
 
 @pytest.fixture()
-def sushi_no_default_catalog(mocker: MockerFixture) -> Context:
+def sushi_no_default_catalog(mocker: MockerFixture, init_and_plan_context: t.Callable) -> Context:
     mocker.patch(
         "sqlmesh.core.engine_adapter.base.EngineAdapter.default_catalog",
         PropertyMock(return_value=None),
     )
-    context, plan = init_and_plan_context("examples/sushi", mocker)
+    context, plan = init_and_plan_context("examples/sushi")
     assert context.default_catalog is None
     context.apply(plan)
     return context
 
 
-def init_and_plan_context(
-    paths: str | t.List[str],
-    mocker: MockerFixture,
-    config="test_config",
-) -> t.Tuple[Context, Plan]:
-    delete_cache(paths)
-    sushi_context = Context(paths=paths, config=config)
-    confirm = mocker.patch("sqlmesh.core.console.Confirm")
-    confirm.ask.return_value = False
+@pytest.fixture
+def init_and_plan_context(copy_to_temp_path, mocker) -> t.Callable:
```

Comment:
Is `copy_to_temp_path` used here?

Reply:
Thanks that is a mistake when refactoring before opening PR